dataactbroker/fileRoutes.py

```diff
@@ -65,9 +65,28 @@ def list_submissions():
         limit = request.args.get('limit')
         certified = request.args.get('certified')
 
-        accountManager = AccountHandler(request,bcrypt = bcrypt)
-
-        return RouteUtils.run_instance_function(accountManager, accountManager.list_submissions, page, limit, certified)
+        # convert params and type check
+        try:
+            page = int(page) if page is not None else 1
+        except:
+            raise ValueError("Incorrect type specified for 'page'. Please enter a positive number.")
+
+        try:
+            limit = int(limit) if limit is not None else 5
+        except:
+            raise ValueError("Incorrect type specified for 'limit'. Please enter a positive number.")
+
+        if certified is not None:
+            certified = certified.lower()
+        else:
+            raise ValueError("Missing required parameter 'certified'")
+        # If certified is none, get all submissions without filtering
+        if certified is not None and certified not in ['mixed', 'true', 'false']:
+            raise ValueError("Incorrect value specified for the 'certified' parameter")
```

Comment:
I'm all for moving this user-error handling here, but I think that means we need to handle the `ValueError`s differently. `RouteUtils.run_instance_function` is where that crazy try-catch logic is currently happening. Maybe this is a good time to add a [flask-style exception handler](http://flask.pocoo.org/docs/0.11/patterns/apierrors/)?